migrations/0000_createDbs.sql

```diff
@@ -1,26 +1,31 @@
 -- Types
-CREATE TYPE note_type AS ENUM ('predictions', 'marginalia', 'meta', 'questions');
+CREATE TYPE note_type AS ENUM ('uncategorized', 'predictions', 'marginalia', 'meta', 'questions');
```

Comment:
I think it might make more sense to have another relation table for note_type instead of adding this category

Reply:
Having a table that relates note_type and note seems like it assumes the wrong things. The one thing i like about it being in the row itself is that its clear that a note can only have one type. Which i know you can also do with sql restrictions, but it feels cleaner. 

I do understand that to keep your vision pure about making notes immutable, it would have to be referenced since this is an allowed change. However, i'm not i fully agree with that vision. At least not right now. It might be something that we want, but i think we should cross that bridge when we get there and only make it a requirement now if we know we will add that feature later.

Reply:
I'm not sure how you can have those reservations about noteType but not about publication. Why should they be treated differently?